assets/terraform/gce/output.tf

```diff
@@ -0,0 +1,20 @@
+#
+# Output Variables
+#
+
+output "private_ips" {
+  value = ["${google_compute_instance.node.*.network_interface.0.address}"]
+}
+
+output "public_ips" {
+  value = ["${google_compute_instance.node.*.network_interface.0.access_config.0.assigned_nat_ip}"]
+}
+
+# output "pod_cidr_blocks" {
```

Comment:
I want to keep these (and all related commented out block elsewhere) as these can be used to our advantage when there's a fannel alternative as we can save some route quotas.